keys/employees-keys/README.md

```diff
@@ -49,9 +66,9 @@ Please specify how long the key should be valid.
       <n>w = key expires in n weeks
       <n>m = key expires in n months
       <n>y = key expires in n years
-@Key is valid for? (0) 0
-Key does not expire at all
-@Is this correct? (y/N) y
+Key is valid for? (0) 2y
+Key expires at Fri 07 Apr 2023 11:40:33 AM CEST
```

Comment:
1) What to do when this key expires?
2) If a previously arranged key is temporarily inaccessible _(i.e. you're away from a PC with it)_ or has been lost?
- Just make a new key then?

Reply:
@mikebdp2

1. There are 2 possible situations either it just expires, then you should simply extend the validity timeline as described [here](https://unix.stackexchange.com/a/177310). If use extension, then why we just not set an unlimited timeline? Because people forget about if someone extends expiry time it means the key is still in use. Another problem is if you lost a private key or you consider it is compromised, that means you should revoke it. Procedure is [here](https://superuser.com/a/1526287)
2. You should extend the expiration time before it expires, or you can do that after, but there would be some gap when the key would be not valid.

Both questions are valid, so I will add this discussion to the document.